stacks/container-apps/augury-alarms.yml

```diff
@@ -0,0 +1,46 @@
+# stacks/container-apps/augury-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Augury alarms
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AuguryWorkerLogGroupName:
+    Type: String
+Resources:
+  AuguryActualsIngestJobInvocationMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '[ImpressionsFetchJob] Success'
```

Comment:
Are there any other patterns to track? Failures, maybe? Or any specific values coming out of each job?